.github/workflows/create_change_set_workflow.yml

```diff
@@ -0,0 +1,19 @@
+name: Create Change Set
+
+on:
+  pull_request:
+    types: [synchronize, opened, edited]
+    paths-ignore:
+      - 'changelogs/fragments/**/*'
+
+jobs:
+  update-changelog:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v4
+      - name: Update Changelog
+        uses: BigSamu/OpenSearch_Change_Set_Create_Action@main
```

Comment:
Should we move the action repo under opensearch-project? @ashwin-pc

Reply:
We can in future, but lets first start using it and see how effective it is. Once we have more repo's adopting this we can start that process. I dont want to hold this PR for just that reason.

Reply:
#### Security Issue:
This GitHub action could be updated to inject unexpected content into this repo's workflows, see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions 

Fix is to publish v1 of OpenSearch_Change_Set_Create_Action and switch to that tag.